packages/nextjs/components/ExampleUi/ContractInteraction/PurposeSetter.tsx

```diff
@@ -0,0 +1,29 @@
+import { ArrowSmallRightIcon } from "@heroicons/react/24/outline";
+
+export const PurposeSetter = () => {
+  return (
+    <div className="flex flex-col mt-6 px-7 py-8 bg-base-200 opacity-80 rounded-2xl shadow-lg border-2 border-black">
+      <span className="text-6xl text-black">Set a Purpose_</span>
+
+      <div className="mt-8 flex gap-5 items-center">
+        <input
+          type="text"
+          placeholder="Write your purpose here"
+          className="input w-full px-5 bg-gradient bg-[length:100%_100%] border border-black text-2xl placeholder-white"
+        />
+        <div className="flex rounded-full border border-black p-1 flex-shrink-0">
+          <div className="flex rounded-full border-2 border-black p-1">
```

Comment:
`border-primary` instead of `border-black`?

Reply:
it's black on designs :)

Reply:
`border-primary` will be black, right? As defined on the `exampleUi` theme in `tailwind.config.js`.

It's not a big deal, but I try to use theme colors instead of hardcoded ones.